stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/search/SearchView.kt

```diff
@@ -40,12 +49,16 @@ public class SearchView : FrameLayout {
 
         binding.inputField.doAfterTextChanged { newText ->
             updateClearButtonVisibility(newText)
-            listener?.onInputChanged(query)
+            val newQuery = query
```

Comment:
Using query instead of newText because it newText is not trimmed?

Reply:
Yeah, figured this is more useful to output from the view